compiler/z/codegen/OMRTreeEvaluator.hpp

```diff
@@ -687,6 +687,69 @@ class OMR_EXTENSIBLE TreeEvaluator: public OMR::TreeEvaluator
 
    private:
 
+   /** \brief
```

Comment:
@fjeremic Changes in Z Codegen looks fine to me. Comments we have on SymbolReferenceTable makes it really easy to understand what we are inlining. Is it possible to have similar small explanation here as well, Just makes it easier to understand the code.

Reply:
Fixed in 9b514d3